### PR TITLE
Created pitercss_conf-2017

### PR DESCRIPTION
**Pull request by ionakathryn**
Added to events, also added Github value and key to json file

### CODE REVIEW

File: data/events/pitercss_conf-2017 [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    "languages": [
+        "English",
+        "Russian"   
+    ],
+    "location": {
+        "city": "Saint Petersburg",
+        "country": "Russia",
+        "state": ""
+    },
+    "name": "PiterCSS Conference-2017",
```
Comment (rollandf) [inline data/events/pitercss_conf-2017]: No need to have the year in the name.

**Comment by rollandf** [comment]
@ionakathryn the file neam should end with .json
Also use '-' instead of '_' : pitercess-conf-2017.json

Check on the readme how to check that your file is OK before pushing again.

File: data/events/pitercss_conf-2017 (hunk 1 above)
Comment (ionakathryn) [reply data/events/pitercss_conf-2017]: May be worth changing the docs/EVENTS.md file as it says:

"The name might need to include the country/city and the year. Check similar events.  It should end with the year of the event. (e.g. 2016)"

Comment (rollandf) [reply data/events/pitercss_conf-2017]: Sure, you can send a PR to fix the EVENTS.md

**Comment by rollandf** [comment]
👍 Thanks

**Comment by szabgab** [comment]
thanks

**Comment by szabgab** [comment]
Posed here: https://codeandtalk.com/e/pitercss-conf-2017

**Pull request merged**